### PR TITLE
Added u128 division

### PR DESCRIPTION
**Pull request by javiersuweijie**

### CODE REVIEW

File: assembly/integer/u128.ts [hunk 1]
```diff
@@ -463,11 +462,11 @@ export class u128 {
   }
 
   @inline @operator('%')
-  static rem(a: u128, b: u128): u128 {
```
Comment (MaxGraey) [inline assembly/integer/u128.ts]: I think you should cast result to `u128` anyway. Like: `return u128.from(__divmod_rem)` or `return new u128(__divmod_rem)`

**Comment by MaxGraey** [comment]
Also will be great add test for division by zero like:

```ts
export class u128ThrowableTests {
  function shouldThrowDivByZero(): void {
    var a = u128(1);
    var b = u128.Zero;
    a / b;
  }
}
```
```ts
test("Should throw div by zero", t => {
  t.throws(() => shouldThrowDivByZero());
});
```
May be in separate file?

**Comment by MaxGraey** [comment]
Thanks!
